common/common.cpp

```diff
@@ -1700,7 +1700,8 @@ void dump_kv_cache_view_seqs(const llama_kv_cache_view & view, int row_size) {
             if (cs_curr[j] < 0) { continue; }
             if (seqs.find(cs_curr[j]) == seqs.end()) {
                 if (seqs.size() + 1 >= sizeof(slot_chars)) { break; }
-                seqs[cs_curr[j]] = seqs.size();
+                auto current_size = seqs.size();
+                seqs[cs_curr[j]] = current_size;
```

Comment:
There's an undefined behaviour related to the fact that the order of the calculation is not defined. It would either call `seqs.size()` at first, or will create a key-value pair in the map, which will affect the size. Here's a great explanation: https://pvs-studio.com/en/docs/warnings/v708/

Reply:
I see. You should add a comment to make it clear why this is necessary.